### DISCUSSION TITLE
sound@cinnamon.org - Mini Play/Pause icon, instead of musical note icon, during playback

### DISCUSSION THREAD

**Discussion by lukaszpolowczyk** [linuxmint/discussions#507] [Ideas]
The music note icon bothers me because it replaces the speaker icon and there is no graphical volume and mute indicator.
The musical note icon also disturbs my sense of aesthetics - such a complete replacement of the icon.

I suggest creating a mini play/pause icon that would be displayed, for example, in the lower left corner of the speaker icon.

Something like this (I'm not a graphic designer, it should be clearer):
![obraz](https://github.com/linuxmint/discussions/assets/16800535/1c439c28-8206-437f-8d64-b8708507b855)
And similarly with Pause.
 
What do you think?

**Comment by claudiux** [comment]
Interesting. I'll try to implement that in the `sound150@claudiux` applet (which can already display volume level near its icon): https://cinnamon-spices.linuxmint.com/applets/view/306
